### ISSUE TITLE
Chobury - Democratic City for All

### ISSUE THREAD

**Issue by QuillzLouis** [decentraland/districts#55]
# Welcome to my District Proposal
By popular request I have updated the proposal, to provide more specific details and provide more of an outline to the district.

### Basic Description
Chobury is a district with areas with apartment's and other high density personal property, leading into lots of commercial business areas, which will sell and provide services.

### How will the district be run?
The district will be run as a democratic state, we are still up for discussion on the type of democracy, but currently we are looking at a direct democracy, where the people directly vote on everything that the city will do, without democratic leaders.

##### Project Leader:
Louis Allaway
rocket.chat handle: @louis.allaway
Twitter: https://twitter.com/chobury_dcl

### Hmmm... Whats the Point?
The goal of this district is to provide a place for people to enjoy themselves. Current ideas being an underground Club/Bar, where we have a DJ playing music and run events, such as an open mic night, talent shows, quiz nights and just general socialising. We are also going to provide areas for modern shops, for people to browse around and enjoy. These stores will have many experiences you just couldn't get while browsing online.

###### Catagory:
           Public
###### Land Needed:
           As much as possible
###### Startup Costs:
           It is a goal of the project to outsource some scripting or 3D Modelling, so money would be nice, but we will be able to get by doing it ourselves/volunteers.
###### How will funds be distributed:
           I am working on a new pie chart. However I am looking to have a 5% to founding team 25% to original stakers and 70% back into the city, however all these numbers will probably change.
###### How will contributes see a return value:
           They will own some land in the city and be citizens of the city and lifetime city membership and 25% of profits.
###### How will the city gain profits
- Selling/renting property
- Rent
- City Memberships (For non residents to get extra city functions like ad free.)
- Digital Dynamic Advertisements (Dm me for more information)

#### Idea for center of town building:
![Front of Grand Centeral](https://user-images.githubusercontent.com/16400169/32110848-8bfc36b2-bb30-11e7-8938-3d86bfc6020f.jpg)

![](https://i2-prod.birminghammail.co.uk/incoming/article9625495.ece/ALTERNATES/s615/291014_high-res-interior-atrium-LATEST-1711-1.jpg)

#### Modern apartment building
![](https://s-media-cache-ak0.pinimg.com/originals/5c/8d/af/5c8daf6f9caecf0402d947ea115bf8f3.jpg)

#### Modern Store Front 
Yes I understand we all hate apple, but you have got to admit some of the stores look nice. Coming from a stern apple hater.
![](https://images.apple.com/cn/retail/nanjingjinmaoplace/images/hero_large.jpg)

#### Bar/Club/Expo Center idea
Again we all hate apple, but I like the idea of the underground facility with the sloping down entrance, and it allows to maximize social space.
![](https://images.adsttc.com/media/images/5928/5e68/e58e/ce97/f600/00f4/newsletter/gallery_image1_large_2x.jpg)

**Comment by QuillzLouis** [comment]
Updated Democracy Proposal, open to suggestions on the issue.

**Comment by QuillzLouis** [comment]
Added Communication Channels

**Comment by QuillzLouis** [comment]
Updated City Profit Distribution Chart

**Comment by uvizhe** [comment]
Can you please clarify what is the difference between residents and investors? Who they are?

**Comment by QuillzLouis** [comment]
@uvizhe Residents are people that live inside the city, and investors are people who donate/contribute to the city.

**Comment by uvizhe** [comment]
@QuillzLouis what do you mean by living in a VR-city?

**Comment by QuillzLouis** [comment]
@uvizhe As in people who own/rent property in the area. This is mostly going to be buisness etc.

**Comment by Dmitry1987** [comment]
@uvizhe those gamers who are too lazy to get out to a real pub lol

**Comment by Dmitry1987** [comment]
but with enough games and quizzes this can be interesting.  I see no point in "business buildings", because who wants to go to work in VR :D  but it has a point in regards of virtual disco and party games, especially with hands\legs controllers that will replicate your real movements to the avatar 3D model that you control inside the game. and everyone can see each other moves.  maybe in year 2025 this will be implemented and working fluently. why not :)

**Comment by seanlynch** [comment]
Don't discount working in VR. Spatialized audio, assuming DCL supports it, is vastly superior to conventional teleconferencing technologies. When I worked at Linden Lab we'd have our meetings in Second Life even when most of us were in the same room together, because the experience was so much better for people joining remotely, and it meant we didn't need so many conference rooms. Dev teams would sit in Second Life while they were working so they could walk over and talk to people even if they weren't in the same city.

I actually think remote work is VR's "killer app". Probably the main thing still missing is mapping someone's facial pose onto the avatar, since the fake lipsync Second Life has is somewhat disconcerting and nuance is still lost if you can't see someone's face. Even a floating window as the avatar's face might be an improvement over an avatar using fake animations.

**Comment by seanlynch** [comment]
Also, a lot of the people I've run into in Second Life are sociophobes or otherwise have some challenge with getting out and socializing. I would definitely not characterize any of them as "too lazy to get out to a real pub." People who have a preference and ability for in-person interaction tend to do that.

Also, it's much easier to seek out like-minded folks in a virtual world than it is in a pub unless you get really lucky and happen to like the particular crowd that hangs out at your local pub. But people for whom that's likely to be the case tend not to be the types who use virtual worlds in the first place.

**Comment by Dmitry1987** [comment]
@seanlynch  very good points. Second Life had this 'Spatialized audio' thing?  you could talk to people and walk around?

**Comment by seanlynch** [comment]
Yes, though they were using a proprietary service called Vivox to do the mixing, because it mixes the audio streams server-side using only the position and then handles head rotation client side using a single stream. Not sure if there's an open source solution that does this yet, but I'm also not sure sending multiple audio streams to each client is that big of a deal either.

**Comment by seanlynch** [comment]
Not sure if Vivox uses it, but [Ambisonics](https://en.wikipedia.org/wiki/Ambisonics) looks promising. There's a bunch of open source software implementing it listed [here](https://en.wikipedia.org/wiki/List_of_Ambisonic_software).

**Comment by Dmitry1987** [comment]
cool. but first they'll need to implement multiplayer at all, so it'll take time...  we probably will see something worth playing within a year, not earlier

**Comment by Dmitry1987** [comment]
and something worth "**paying**" within 2-3 years i guess.

**Comment by Dmitry1987** [comment]
So don't expect guys to "rent your land" for hundreds $ in near future :D seriously. From all what I've read until now, all DCL community is so delusional. Not a single person realizing the difficulties and what market realities are.
Who will stop real giants like Crytek, Blizzard, etc' from creating a quality VR world of their own, if this idea will be successful? And who from current DCL community can compete with those in terms of quality content?

**Comment by Dmitry1987** [comment]
Btw when I asked how "decentralized" it really is, and whether I can create a pokemon game inside it - I was told Nintendo will shut down DCL :+1:  so it'll not be allowed, nice "decentralized" metaverse lol

**Comment by seanlynch** [comment]
I'm not sure you can have something that's truly decentralized if there's a company controlling it. I think you need to do something like what BitTorrent, Inc is doing, where they're not trying to make money off the BitTorrent network itself, but instead just making software.

I guess the question is going to be whether it's possible to use DCL's software to create worlds that aren't disconnected to DCL the company in any way without having to hack it. If not, then I'm glad I only put a little money in, because we don't need yet another censored virtual world.

**Comment by seanlynch** [comment]
If I had waaaaay more free time than I do, I'd be looking into porting one of the open source Second Life viewers to Emscripten, possibly via Oryol 3D, and building a WebSockets to SL protocol proxy and using OpenSim, or just building WebSockets support into OpenSim itself.

**Comment by QuillzLouis** [comment]
Working on a new plan :)

**Comment by QuillzLouis** [comment]
Hello.
I do confirm I am @QuillzToxic#8653 on Discord.

**Comment by scottie** [comment]
Is this town dead ?

**Comment by scottie** [comment]
https://discord.gg/sKJTN3J
